reframe/core/launchers/mpi.py

```diff
@@ -124,3 +124,26 @@ def command(self, job):
             ret.append(opt)
 
         return ret
+
+
+@register_launcher('lrun')
+class LrunLauncher(JobLauncher):
+    '''LLNL's custom parallel job launcher.'''
+    def command(self, job):
+        num_tasks = job.num_tasks or 1
```

Comment:
I'm not sure that this is needed. `job.num_tasks` is always > 0. If it is set to zero, then the scheduler will try to do a flexible allocation if supported.

Reply:
done